diktat-rules/src/test/resources/test/paragraph2/kdoc/ConstructorCommentPropertiesTest.kt

```diff
@@ -185,18 +183,18 @@ open class B<K : Any> constructor(
  * @property openAddr override
  *   property info
  */
-class A<K : Any, P: Any, G: Any> constructor(
-    //single-line comment
+class A<K : Any, P : Any, G : Any> constructor(
+    // single-line comment
```

Comment:
I thought that test was checking how this space is fixed

Reply:
No, this test checks how comments on properties in the primary constructor are added to existing `@property`-s in KDoc